### PR TITLE
AT_01.12.02 | Start page > Register Agent Account Popup > Footer UI and functionality > Insure Footer elements are visible

### PR DESCRIPTION
**Pull request by Massik78**
[[AT_01.12.02 | Start page > Register Agent Account Popup > Footer UI and functionality > Insure Footer elements are visible](https://trello.com/c/aleMsnIg/305-at011202-name)](https://trello.com/c/aleMsnIg/305-at011202-name)

### CODE REVIEW

**Comment by Massik78** [comment]
[AT_01.12.02 | Start page > Register Agent Account Popup > Footer UI and functionality > Insure Footer elements are visible](https://trello.com/c/aleMsnIg/305-at011202-name)

**Comment by Massik78** [comment]
[AT_01.12.02 | Start page > Register Agent Account Popup > Footer UI and functionality > Insure Footer elements are visible](https://trello.com/c/aleMsnIg/305-at011202-name)

File: cypress/e2e/testUiAndFunction/startPage/registerAgentAccountPopupSection/US_01.12_FooterUIandFunc.cy.js [hunk 1]
```diff
@@ -22,4 +22,10 @@ describe('US_01.12 | Footer UI and functionality', () => {
         registerPopup.getRegisterAgentAccountHeader().should('not.be.visible')
         restorePopup.getRestorePasswordHeader().should('be.visible')
     })
+
+    it('AT_01.12.02 | Verify Footer elements are visible' , () => {
+        //startPage.getRegisterAccountLink();
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/registerAgentAccountPopupSection/US_01.12_FooterUIandFunc.cy.js:27]: Please remove the comment from the code

File: cypress/e2e/testUiAndFunction/startPage/registerAgentAccountPopupSection/US_01.12_FooterUIandFunc.cy.js [hunk 2]
```diff
@@ -22,4 +22,10 @@ describe('US_01.12 | Footer UI and functionality', () => {
...
+        startPage.clickRegisterAccountLink();
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/registerAgentAccountPopupSection/US_01.12_FooterUIandFunc.cy.js:28]: Please move lines 28 and 20 to beforeEach, as this is a precondition and will be executed in all tests of this user story.
Note: Line 20 is misspelled, it should be a click on the element's link (startPage.clickRegisterAccountLink();), not a get of the element itself.

**Pull request closed without merging**